hedera-node/hedera-app/src/main/java/com/hedera/node/app/authorization/PrivilegesVerifier.java

```diff
@@ -83,9 +86,21 @@ public SystemPrivilege hasPrivileges(
                     payerId, txBody.fileUpdateOrThrow().fileIDOrThrow().fileNum());
             case FILE_APPEND -> checkFileChange(
                     payerId, txBody.fileAppendOrThrow().fileIDOrThrow().fileNum());
-            case CONTRACT_UPDATE -> checkFileChange(
-                    payerId,
-                    txBody.contractUpdateInstanceOrThrow().contractIDOrThrow().contractNumOrThrow());
+            case CONTRACT_UPDATE -> {
+                var contractId = txBody.contractUpdateInstanceOrThrow().contractIDOrThrow();
+
+                if (contractId.hasEvmAddress()) {
```

Comment:
There are no system contracts (below `0.0.1000`) to check for special update and delete privileges---and almost certainly never will be such system contracts.

(If there _were_, I don't think we would want to assume they only had long-zero EVM addresses.)

I'd suggest just deleting the `CONTRACT_UPDATE` and `CONTRACT_DELETE` branches in this switch; created https://github.com/hashgraph/hedera-services/issues/9383 for that work since all checks are passing here.